jobs/tendrl-jobs-build-triggers.yaml

```diff
@@ -0,0 +1,77 @@
+- job-template:
+    name: "tendrl-build-{package_name}"
+    display-name: 'Tendrl-{package_name}'
+    description: 'Do not edit this job through the web!'
+    project-type: freestyle
+    defaults: global
+    disabled: false
+    concurrent: false
+    quiet-period: 0
+    block-downstream: false
+    block-upstream: false
+    node: 'master'
+    triggers:
+        - timed: 'H H(0-7) * * *'
+        - pollscm:
+            cron: 'H/10 * * * *'
+            ignore-post-commit-hooks: false
```

Comment:
I think you only need `pollscm` here, right? `timed` means it's building the same thing over and over